soc/intel/intel_niosv/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Nordic Semiconductor
+# Copyright (c) 2023-2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# TODO: adjust when migrated
+add_subdirectory(../../soc_legacy/riscv/common common)
```

Comment:
Have moved this as per https://github.com/zephyrproject-rtos/zephyr/pull/68872 but need to wait for feedback to ensure it is acceptable (marking as NACK to prevent merge)

Reply:
changed